build.sbt

```diff
@@ -15,33 +14,34 @@ def scalaBinaryVersion(scalaVersion: String) = scalaVersion match {
   case version if version startsWith "2.12" => "2.12"
   case _ => sys.error("unknown error")
 }
-def isScala210x(scalaVersion: String) = scalaBinaryVersion(scalaVersion) == "2.10"
 
 val algebirdVersion = "0.13.0"
 val apacheCommonsVersion = "2.2"
 val avroVersion = "1.7.4"
-val bijectionVersion = "0.9.1"
+val bijectionVersion = "0.9.5"
 val cascadingAvroVersion = "2.1.2"
-val chillVersion = "0.8.3"
+val chillVersion = "0.8.4"
 val elephantbirdVersion = "4.8"
 val hadoopLzoVersion = "0.4.19"
 val hadoopVersion = "2.5.0"
 val hbaseVersion = "0.94.10"
 val hravenVersion = "0.9.17.t05"
-val jacksonVersion = "2.4.2"
-val json4SVersion = "3.2.11"
+val jacksonVersion = "2.8.7"
+val json4SVersion = "3.5.0"
 val paradiseVersion = "2.1.0"
 val parquetVersion = "1.8.1"
 val protobufVersion = "2.4.1"
-val quasiquotesVersion = "2.0.1"
-val scalaCheckVersion = "1.12.2"
-val scalaTestVersion = "2.2.6"
-val scalameterVersion = "0.6"
-val scroogeVersion = "3.20.0"
+val scalameterVersion = "0.8.2"
+val scalaCheckVersion = "1.13.4"
+val scalaTestVersion = "3.0.1"
+val scroogeVersion = "4.12.0"
 val slf4jVersion = "1.6.6"
 val thriftVersion = "0.5.0"
 val junitVersion = "4.10"
 val macroCompatVersion = "1.1.1"
+val jlineVersion = "2.14.3"
+
+def scrooge(scalaVersion: String) = "com.twitter" %% "scrooge-serializer" % scroogeVersion
```

Comment:
I think that was old stuff from when we were still supporting scala 2.10 (which scrooge stopped supporting).